### ISSUE TITLE
Lattice Parser Seg Fault

### ISSUE THREAD

**Issue by dylan171** [frib-high-level-controls/FLAME#16]
In an interactive session accidentally called the parse() method without any arguments and got a seg fault:

```bash
Python 2.7.9 (default, Mar  1 2015, 12:57:24) 
[GCC 4.9.2] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import flame
>>> flame.GLPSParser().parse()
Segmentation fault
```

**Comment by mdavidsaver** [comment]
Should be fixed with dd0174b12405cb50412d8d5a4206299feb15b0fe

**Comment by dylan171** [comment]
That fixed it. Thanks.
